Assets/BossRoom/Prefabs/UI/BossRoomHudCanvas.prefab

```diff
@@ -1286,7 +1286,7 @@ RectTransform:
   m_Children:
```

Comment:
Just a general nitpicky cleanup comment: for UI elements, keep "Raycast Target" enabled only for the necessary components. Some text items, and nested images can have that field disabled. TextMeshPro components have these hidden under "Extra Settings" btw.